src/services/domainService.js

```diff
@@ -217,9 +219,8 @@ async function generateAndReplaceMainApplicationHaproxyConfig(isGmode = false, t
   try {
     if (isGmode) {
       log.info(`G Mode STARTED at${new Date()}`);
-      if (!recentlyConfiguredApps) {
-        throw new Error('G Mode is awaiting processing');
```

Comment:
but this is important. Without it on first reload we would just put only G apps to the haproxy deleting other apps

Reply:
ahhhhhhhh makes sense, I forgot about it.

Reply:
will take care of it

Reply:
It's fixed plus some more improvements